webrtc/sdp-offer-dc.html

```diff
@@ -0,0 +1,110 @@
+<!doctype html>
+<!--
+This test uses data only, and thus does not require fake media devices.
+-->
+
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>SDP generated by createOffer</title>
+</head>
+<body>
+  <div id="log"></div>
+
+  <!-- These files are in place when executing on W3C. -->
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/vendor-prefix.js"
+           data-prefixed-objects=
+        '[{"ancestors":["window"], "name":"RTCPeerConnection"}]'></script>
+  <script type="text/javascript">
+  var test = async_test('The SDP generated by createOffer matches JSEP requirements for initial offer.');
```

Comment:
Spec link?
This seems to be testing JSEP. Should we put JSEP tests in a separate subdirectory of webrtc?
With JSEP, it makes sense to have a spec link with the draft number in it.

Reply:
Given JSEP is now IESG approved, I hope there are not significant changes in upcoming version.